gammapy/catalog/tests/test_hawc.py

```diff
@@ -0,0 +1,27 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import pytest
+from ..hawc import SourceCatalog2HWC
+from ...utils.testing import requires_data
+
+@pytest.fixture(scope='session')
+def hwc2_fhl():
+    return SourceCatalog2HWC(filename = filename) 
+
+@requires_data('gammapy-extra')
+class TestSourceCatalog2HWC:
+    def test_source_table(self, hwc2_fhl):
+        assert hwc2_fhl.name == '2hwc'
+        assert len(hwc2_fhl.table) == 40
+
+@requires_data('gammapy-extra')
+class TestSourceCatalogObject2HWC:
+    def test_data(self, hwc2_fhl):
+        source = hwc2_fhl[0]
+
+        assert source.data['source_name'] == '2HWC J0534+220'
+
+    #TODO: Add this @pytest.mark.parametrize thing?
```

Comment:
I will write a test method here but can you tell me what I have to write into this @pytest.mark.parametrize thing.
Thx

Reply:
To learn how `pytest` works, please read this: https://github.com/jiffyclub/pytest-features#parametrized-tests or copy / execute the other working examples in `gammapy.catalog`. You can use it if you like, but I'm not really sure if it's useful here to parametrize tests. I would probably not do it and just have one test for a single-spec source and one separate test for a source with 2 spectra.